docs/storm-sql-reference.md

```diff
@@ -189,7 +189,7 @@ may refer to tables in the FROM clause of an enclosing query.
 
 *selectWithoutFrom* is equivalent to VALUES,
 but is not standard SQL and is only allowed in certain
-[conformance levels]({{ site.apiRoot }}/org/apache/calcite/sql/validate/SqlConformance.html#isFromRequired--).
+[conformance levels](https://calcite.apache.org/apidocs/org/apache/calcite/sql/validate/SqlConformance.html#isFromRequired--).
```

Comment:
Can we avoid hard-coding the host?

Reply:
In other instances, there is just a link beginning with `javadocs` leading to the local copy of generated javadoc (e.g., to http://storm.apache.org/releases/1.2.2/javadocs/... ), but `org.apache.calcite.sql` is not included there. So I don't think so.

Reply:
Right, makes sense.